JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -4,9 +4,40 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
-
-// 과일 저장소 타입
 class FruitStore {
+    private var stock: [Fruits: Int] = [.strawberry: 10, .banana: 10, .pineapple: 10, .kiwi: 10, .mango: 10]
+    
+    private func hasFruit(menu: Menu, total: Int) -> Bool {
+        let necessaryFruit = menu.count(to: total, of: menu)
+        
+        for (fruit, need) in necessaryFruit {
+            guard let number = stock[fruit],
+                    number >= need
+            else {
+                return false
+            }
```

Comment:
for 문에 guard 의 else 문에만 코드가 있고
걸리지고 난 뒤에는 코드가 없어서 보기가 조금은 어려워 보이네요
return false 에 대한 코드가 else 가 아닌 guard 에서 걸리진 곳에서 보이는 게 다른 사람들이 보기에 더 잘 보일듯 합니다.

```suggestion
    guard let number = stock[fruit] else { continue }
    if number < need {
        return false 
    }        
```

Reply:
훨씬 이해하기 편하네요 감사합니다!